Lombiq.DataTables/Controllers/TableController.cs

```diff
@@ -9,7 +9,6 @@
 
 namespace Lombiq.DataTables.Controllers;
 
-[Admin]
```

Comment:
Why is the removal of this necessary? This is a security change.

Reply:
The [action](https://github.com/Lombiq/Orchard-Data-Tables/pull/165/files/fa7c86da4d5cce4c1312afd9451c514d581bcea6#diff-b585eaaeee4be5bea53d9863204af1b5cb39205bc4eac1cf6997e21942365981R19) already has it (where it's necessary for the path), so this is redundant.

Reply:
Oh OK.